pages/common/git-push.md

```diff
@@ -17,3 +17,7 @@
 - Remove remote branches which don't exist locally
 
 `git push --prune {{REMOTE-NAME}}`
+
+- Publish tags
```

Comment:
I meant all the examples in this page should use the same terminology to avoid confusion. So in this case, either use the "publish" term in the other examples as well, or not use it in the tags example.